apps/dashboard/src/components/primitives/input.tsx

```diff
@@ -5,7 +5,37 @@ import { cva, VariantProps } from 'class-variance-authority';
 import { inputVariants } from '@/components/primitives/variants';
 
 const inputFieldVariants = cva(
-  'text-foreground-950 flex w-full flex-nowrap items-center gap-1.5 rounded-md border bg-transparent transition-colors focus-within:outline-none focus-visible:outline-none hover:bg-neutral-50 has-[input:disabled]:cursor-not-allowed has-[input:disabled]:opacity-50 has-[input[value=""]]:text-foreground-400 has-[input:disabled]:bg-neutral-alpha-100 has-[input:disabled]:text-foreground-300',
+  [
+    // Base styles
+    'text-foreground-950',
+    'flex w-full flex-nowrap',
+    'items-center gap-1.5',
+    'rounded-md border',
+    'bg-transparent',
+    'transition-colors',
+
+    // Focus states
+    'focus-within:outline-none',
+    'focus-visible:outline-none',
+
+    // Hover state
+    'hover:bg-neutral-50',
+
+    // Disabled state
+    'has-[input:disabled]:cursor-not-allowed',
+    'has-[input:disabled]:text-foreground-300',
+    'has-[input:disabled]:bg-neutral-alpha-100',
+    'has-[input:disabled]:opacity-50',
+
+    // Empty value state
+    'has-[input[value=""]]:text-foreground-400',
+
+    // Read-only state
+    'has-[input:read-only]:text-foreground-700',
+    'has-[input:read-only]:bg-neutral-alpha-100',
+    'has-[input:read-only]:opacity-70',
+    'has-[input:read-only]:border-neutral-alpha-200',
+  ].join(' '),
```

Comment:
This format is more readable, not sure if it can have some unintended consequences by not being static but using join

Reply:
are these styles agreed with Naveen? because I don't see the design system updated

Reply:
Just [confirmed with Naveen](https://linear.app/novu/issue/NV-4637/apply-styling-fixes-to-workflow-identifier-field#comment-9de33e29), should be good.